TombEngine/Game/items.cpp

```diff
@@ -838,61 +838,56 @@ void DefaultItemHit(ItemInfo& target, ItemInfo& source, std::optional<GameVector
 	DoItemHit(&target, damage, isExplosive);
 }
 
-
-//TODO: Originally was using a ItemInfo, but has been simplified to use only vector3i. Is other .cpp more suitable to contain this function?
+// TODO: Originally took an ItemInfo argument, but has been simplified to use only Vector3i.
+// Is other .cpp more suitable to contain this function?
 Vector3i GetNearestSectorCenter(const Vector3i& pos)
 {
-	const unsigned int maskLower8Bits = 0xFFFFFE00;	// mask for the bits [0-8]
-	const unsigned int mask9Bit = 0x200;			// mask for the bit [9] (512)
+	constexpr auto BIT_MASK_LOWER_8 = 0xFFFFFE00; // 0-8
+	constexpr auto BIT_MASK_9		= 0x200;	  // 9
 
-	//Clears the 0-8 bits and set the bit 9 to put 512 at the end. (Placing the object in the sector center).
-	return Vector3i (
-					 pos.x & maskLower8Bits | mask9Bit,
-					 pos.y,
-					 pos.z & maskLower8Bits | mask9Bit
-					);
+	// Return collision block center.
+	// TODO: No bitwise operations.
+	return Vector3i(
+		pos.x & BIT_MASK_LOWER_8 | BIT_MASK_9,
+		pos.y,
+		pos.z & BIT_MASK_LOWER_8 | BIT_MASK_9);
 }
 
-bool CompareItemByXZ(const int itemNumberA, const int itemNumberB)
+bool CompareItem2DPositions(const int itemNumber0, const int itemNumber1)
 {
-	// Compare objects by their XZ position
-	auto& itemA = g_Level.Items[itemNumberA];
-	auto& itemB = g_Level.Items[itemNumberB];
+	const auto& item0 = g_Level.Items[itemNumber0];
+	const auto& item1 = g_Level.Items[itemNumber1];
 
-	if (itemA.Pose.Position.x == itemB.Pose.Position.x)
-	{
-		return itemA.Pose.Position.z < itemB.Pose.Position.z;
-	}
+	if (item0.Pose.Position.x == item1.Pose.Position.x)
+		return (item0.Pose.Position.z < item1.Pose.Position.z);
 	else 
-	{
-		return itemA.Pose.Position.x < itemB.Pose.Position.x;
-	}
+		return (item0.Pose.Position.x < item1.Pose.Position.x);
```

Comment:
This goes against the project coding convention. Lwmte in the civvy branch, asked me to keep the brackets in the if-else and if-elseif structures.

https://github.com/MontyTRC89/TombEngine/issues/294  

> However, if you have only one line enclosed (i. e. only foo), and there is no following else or else if conditions, it is currently allowed (but not recommended) to omit brackets

Reply:
If there are only two conditions, both contain single-line blocks, and the second is an open `else`, I've tended to skip using braces in such instances for some time. Maybe we should get further clarification?